modules/incentives/src/lib.rs

```diff
@@ -337,7 +352,7 @@ pub mod module {
 		#[transactional]
 		pub fn claim_rewards(origin: OriginFor<T>, pool_id: PoolId<T::RelaychainAccountId>) -> DispatchResult {
 			let who = ensure_signed(origin)?;
-			<orml_rewards::Pallet<T>>::claim_rewards(&who, &pool_id);
+			Self::do_claim_reward(&who, &pool_id)?;
```

Comment:
I think we don't have to move dispatchable call impl into a new fn prefixed with `do_` anymore. IDEs are able to do syntax analysis with the new pallet attributes macros.